pkg/kube/client.go

```diff
@@ -361,6 +361,11 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 	err = perform(infos, func(info *resource.Info) error {
 		mux.Lock()
 		defer mux.Unlock()
+		if err := info.Get(); err != nil {
```

Comment:
Calling `info.Get` will refresh the `info.Object` from the api.  This will change the behavior of creating patches during an upgrade.

Reply:
Wait maybe not.  I didn't realize this was only when getting a release.  Let me investigate further.

Reply:
@adamreese any news? This is the behavior that was there before. It was accidentally left off on my part. This restores the previous behavior prior to my series of patches when getting the related pods.

Reply:
We paired on this again, and we _think_ it is okay in this case.

Reply:
Cancel, We are good